docs/use-cases/interoperability/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: XRP Ledger Use Cases
+---
+import { XRPLCardDeck, DeckCard } from "../../../_components/XRPLCardDeck";
+
+# Interoperability
+
+These use cases focus on capabilities on the XRP Ledger to interact with other chains.
+
+<XRPLCardDeck cols="1/3">
+  <DeckCard title="Sidechains (Beta)" to="sidechains.md">
+    Run an independent ledger with its own consensus algorithm and transaction types and rules.
+  </DeckCard>
+  <DeckCard title="XRPL Hooks" to="https://xrpl-hooks.readme.io/" external>
+    Hooks add smart contract functionality to the XRP Ledger: 'layer one' custom code to influence the behaviour and flow of transactions.
```

Comment:
What does it mean to put 'layer one' in single quotes? Should it be double quotes, or is this something in syntax?

Are we using British spelling in our docs (_behavior_)?

Reply:
Good catch. Thanks.